test/ui-test/testSuites/suite_status/shared/steps/chatSteps.py

```diff
@@ -29,6 +33,11 @@ def step(context):
     for row in table[1:]:
         _statusChat.send_message(row[0])
         _statusChat.verify_last_message_sent(row[0])
+        
+@Then("user is able to send a gif message")
+def step(context):
+    _statusChat.send_gif()
+    _statusChat.verify_last_message_sent("DOCTYPE HTML")
```

Comment:
This is not a good check. All messages have this. It would be better to check the url of the gif.
What you can do is move the validation to `send_gif`. Before pressing the Gif, put the url of the gif in a variable and check that the last message contains that url. You will probably need a new objectName to get that url.

Reply:
> This is not a good check. All messages have this. It would be better to check the url of the gif. What you can do is move the validation to `send_gif`. Before pressing the Gif, put the url of the gif in a variable and check that the last message contains that url. You will probably need a new objectName to get that url.

I agree on adding the validation inside `send_gif` and check if contains that url.